spoon-runner/src/test/java/com/squareup/spoon/LogRecordingTestRunListenerTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.squareup.spoon
+
+import com.android.ddmlib.testrunner.TestIdentifier
+import com.squareup.spoon.LogRecordingTestRunListener.Companion.stripParametersInClassName
+import org.hamcrest.CoreMatchers.*
+import org.junit.Assert.assertThat
```

Comment:
let's switch this to use Truth instead (see `DeviceUtilsTest` for example).  Side benefit of consistency with the other tests: no nasty backticks around the names of hamcrest matchers.